SVSModel/ModelInterface.cs

```diff
@@ -28,45 +25,59 @@ public interface IModelInterface
         /// </summary>
         /// <returns>List of <see cref="CropCoefficient"/>s directly from the data file</returns>
         IEnumerable<CropCoefficient> GetCropCoefficients();
 
-        Dictionary<DateTime, double> GetSoilTestResult(DateTime testDate, double testValue, string depthOfSample,
-                                                       string typeOfTest, string moistureOfTest,
-                                                       string categoryOfSoil, string textureOfSoil);
-
+        Dictionary<DateTime, double> GetSoilTestResult(
+            DateTime testDate,
+            double testValue,
+            string depthOfSample,
+            string typeOfTest,
+            string moistureOfTest,
+            string categoryOfSoil,
+            string textureOfSoil);
     }
 
     public class ModelInterface : IModelInterface
     {
-        public Dictionary<DateTime, double> GetSoilTestResult(DateTime testDate, double testValue, string depthOfSample,
-                                                       string typeOfTest, string moistureOfTest,
-                                                       string categoryOfSoil, string textureOfSoil)
+        public Dictionary<DateTime, double> GetSoilTestResult(
+            DateTime testDate,
+            double testValue,
+            string depthOfSample,
+            string typeOfTest,
+            string moistureOfTest,
+            string categoryOfSoil,
+            string textureOfSoil)
         {
-            SoilTestConfig test = new SoilTestConfig(testDate, testValue, depthOfSample,
-                                                       typeOfTest, moistureOfTest,
-                                                       categoryOfSoil, textureOfSoil);
+            var test = new SoilTestConfig(
+                testDate,
+                testValue,
+                depthOfSample,
+                typeOfTest,
+                moistureOfTest,
+                categoryOfSoil,
+                textureOfSoil);
+            
             return test.Result;
         }
 
-        public List<DailyNBalance> GetDailyNBalance(string weatherStation, Dictionary<DateTime, double> testResults, Dictionary<DateTime, double> nApplied, Config config)
+        public DailyNBalanceDTO GetDailyNBalance(string weatherStation, Dictionary<DateTime, double> testResults, Dictionary<DateTime, double> nApplied, Config config)
         {
             var startDate = config.Prior.EstablishDate.AddDays(-1);
             var endDate = config.Following.HarvestDate.AddDays(2);
             var metData = BuildMetDataDictionaries(startDate, endDate, weatherStation, false);
 
-            var rawResult = Simulation.Simulation.SimulateField(metData.MeanT, metData.Rain, metData.MeanPET, testResults, nApplied, config, Constants.InitialN);
+            var rawResult = Simulation.Simulation.SimulateField(metData.MeanT, metData.Rain, metData.MeanPET, testResults, nApplied, config, InitialN);
```

Comment:
We need to check this change to Initial N.  It is important that runs made from the web app are starting with the default initial N

Reply:
I have pulled the Rezere branch and can see what you have done here.  All good